source/command-monitoring/tests/unacknowledgedBulkWrite.json

```diff
@@ -13,6 +13,11 @@
       "comment": "On a 2.4 server, no GLE is sent and requires a client-side manufactured reply",
       "operation": {
         "name": "bulkWrite",
```

Comment:
Where's the "object" field?

Or do we not have consistent formats across tests yet?